core/arch/arm/include/kernel/generic_boot.h

```diff
@@ -50,7 +50,11 @@ extern uint8_t __data_end[];
 extern uint8_t __bss_start[];
 extern uint8_t __bss_end[];
 extern uint8_t __init_start[];
+extern uint8_t __init_end[];
```

Comment:
No, I'll remove it again. I guess I was using it temporary while trying out things.